### PR TITLE
containers/ExplorePage: add a signout button to explore page story #4464

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
added on explore page for now as it's first page you go and means will not disrupt other menus until it is moved into settings

**Comment by phillimorland** [comment]
We should refactor this and put it in it's own file so it can be imported but for now it can be merged

File: containers/Auth/AuthProvider.js [hunk 1]
```diff
@@ -39,6 +39,11 @@ export const AuthProvider = (props) => {
       AsyncStorage.setItem('authToken', authToken);
       setState({ ...state, loading: false, token: authToken });
     },
+    signOut: async () => {
+      console.log('I was triggered during componentDidMount123');
```
Comment (goapunk) [inline containers/Auth/AuthProvider.js:43]: we should remove this and above as well

Comment (Rineee) [reply containers/Auth/AuthProvider.js:43]: Alread did that here https://github.com/liqd/zt-app/pull/78/commits/2d79284b6a28c82a8478682229367eb9080bd96a

**Review by Rineee — APPROVED** [review]
Yeah, really nice!

**Pull request merged**